### PR TITLE
feat: update edit and delete buttons for company cards, #469

### PR DESCRIPTION
**Pull request by tajchumber**
Edit and Delte buttons for company cards are now situated in a dropdown menu in the top right section of the card

(Also contain a bit of cleaning up of the css and js files).

Although I had considered this task completed, just before I committed I realised that the dropdown menu is hidden by the "Hiring" ribbon. See screenshot below:

Screenshot 2019-03-09 at 12 33 11

What shall we do to address this? Any suggestions @doomspork @gemantzu @burden ? :)

### CODE REVIEW

**Comment by zacck** [comment]
Niiiiice I like this version looks cleeaaan

**Comment by doomspork** [comment]
@tajchumber I'll ecto @zacck's feedback: can't we move the dropdown to the left?

**Comment by tajchumber** [comment]
We can move either one, wanted you guys feedback to decide which one! 😊

**Comment by gemantzu** [comment]
I think we should move the hiring on the left and stick the menu on the right. What are we going to do with the options for the jobs?

**Comment by doomspork** [comment]
@tajchumber these changes are awesome. I will defer to @gemantzu and @burden for review 👍

File: assets/css/libs/_ribbon.sass [hunk 1]
```diff
@@ -67,12 +67,12 @@ $ribbon-border: .1rem solid $grey-lighter !default
 
 .corner-top-right-ribbon
```
Comment (lccezinha) [inline assets/css/libs/_ribbon.sass]: @tajchumber rename this css class, now the component will be placed at the left side of the div, so does not make sense anymore the name be `corner-top-RIGHT-ribbon`

File: assets/css/libs/_ribbon.sass [hunk 2]
```diff
@@ -67,12 +67,12 @@ $ribbon-border: .1rem solid $grey-lighter !default
...
   top: -22px
-  right: 25px
+  left: -2px
 
 .corner-top-right-ribbon span
```
Comment (lccezinha) [inline assets/css/libs/_ribbon.sass]: Same goes here.

File: assets/css/libs/_ribbon.sass (hunk 1 above)
Comment (tajchumber) [reply assets/css/libs/_ribbon.sass]: of course! thanks picking this up!

**Comment by doomspork** [comment]
@tajchumber could you resolve the conflict? @gemantzu / @burden / @lccezinha any final thoughts on this?

**Comment by tajchumber** [comment]
@doomspork should be all good to go! waiting for the review 👍

**Pull request merged**